src/commcare_cloud/ansible/roles/postgresql_base/templates/postgresql.conf.j2

```diff
@@ -50,7 +50,7 @@ archive_command = '{{ postgresql_archive_command }}'
 archive_timeout = {{ postgresql_archive_timeout }}
 {% endif %}
 
-wal_level = hot_standby
+wal_level = {{ postgresql_wal_level|default(hot_standby) }}
```

Comment:
`wal_level = logical` is a superset of `hot_standby` according to https://www.postgresql.org/docs/9.6/runtime-config-wal.html